### ISSUE TITLE
Not working in Microsoft Edge, IE11, and Chrome

### ISSUE THREAD

**Issue by cmlee1220** [samwiseduzer/angularPrint#7]
I wasn't sure if you'd get my comment since the issue appeared closed, so i opened this one.  On Microsoft Edge, none of the features work.  For example, the print-remove divs all appear on screen.  On IE, the format doesn't hold on print preview and part of the document gets cut off.  On Chrome, part of the side navbar is added along with a colored block.  In addition, none of the format holds.  

Firefox works beautifully!  Any help would be greatly appreciated!

**Comment by samwiseduzer** [comment]
I'm currently in the process of creating a major release that will add cross-browser compatibility for the vast majority of features. Any non-compatibility will be listed in the documentation. Thanks for creating the issue and thanks for your patience while I resolve it.

**Comment by cmlee1220** [comment]
Sounds great!  I appreciate your work and your reply.

Regards,

Cristina

**Comment by nfasulo** [comment]
I just found this repository and am really excited by the possibility of using it! I was just wondering if there is any update on when we might expect a new release with cross-browser compatibility? Thanks!

**Comment by samwiseduzer** [comment]
Yeah, I'm really sorry for the lack of updates on multi-browser compatibility. I anticipate releasing a new version this weekend that supports chrome, mozilla and safari. Edge will be mostly supported, but may still have a bug or two.

**Comment by adamcee** [comment]
Hi, I just wanted to check on this issue as we are looking at using this. Is there currently support for chrome, firefox/mozilla, safari, and IE 11?

**Comment by andr3i4o** [comment]
Hi, great little peace of magic! I would like to ask you are there any commits coming out soon regarding this issue?

**Comment by ZainabMohd** [comment]
I agree with andr3i4o .. any updates regarding he issue ?

**Comment by varunsharma27** [comment]
I don't think he cares anymore.
